### ISSUE TITLE
Garbage messages

### ISSUE THREAD

**Issue by s3rvant** [rzr/mastodon-lite#5]
Currently messages are only sent when they differ from the previous message. This means that if I want to know when my window opens I need to send another toot to change the message property or else I won't get notified next time the window opens.

Would prefer that after sending the toot the addon property resets to some default so that every time the addon is called webthings will detect a changed property and trigger the toot.

**Comment by rzr** [comment]
Yes I made this to avoid flooding, you could eventually workaround this with timestamp date... 

I assume you're using this module in @mozillaiot gateway ?

Since there is an Alert in schema maybe it could be more appropriate, also this could be aligned to other adapters... 

Meanwhile may I suggest to make this feature optional, I will review your patches ;)
